spec.html

```diff
@@ -5254,6 +5254,18 @@ <h1>CreateDataPropertyOrThrow ( _O_, _P_, _V_ )</h1>
       </emu-note>
     </emu-clause>
 
+    <emu-clause id="sec-createnonenumerabledatapropertyorthrow">
+      <h1>CreateNonEnumerableDataPropertyOrThrow ( _O_, _P_, _V_ )</h1>
+      <p>The abstract operation CreateNonEnumerableDataPropertyOrThrow takes arguments _O_ (an Object), _P_ (a Property Key), and _V_ (an ECMAScript language value). It is used to create a new non-enumerable own property of an object. It throws a *TypeError* exception if the requested property update cannot be performed. It performs the following steps when called:</p>
```

Comment:
```suggestion
      <p>The abstract operation CreateNonEnumerableDataPropertyOrThrow takes arguments _O_ (an Object), _P_ (a property key), and _V_ (an ECMAScript language value). It is used to create a new non-enumerable own property of an object. It throws a *TypeError* exception if the requested property update cannot be performed. It performs the following steps when called:</p>
```

Reply:
This isn’t correct imo; the term is a type and is always capitalized (or should be).

Reply:
Current precedent is that it is not capitalized (none of the existing 88 usages are capitalized, including a number in similar positions to this usage). We could in principle change that, but not as part of this PR.

Reply:
Huh, i could have sworn it was always capitalized, but i can’t find any places where it is, even historically. My mistake.